defs.h

```diff
@@ -91,8 +91,8 @@
 #define PAL16_RED		224
 #define PAL16_GRAY		240
 
-#define SCREEN_WIDTH	640
-#define SCREEN_HEIGHT	480
+#define SCREEN_WIDTH	1280

```

Comment:
Lol you just can't not commit this can you :D

Reply:
XD

Reply:
jesus, how do you handle that? I fail to remember about restoring it every single time lol

Reply:
I commit chunks visually in a GIT GUI (git-cola), do you just hit a bit red button in your IDE?

Reply:
I use commands in bash :D

Reply:
https://desktop.github.com/

https://www.gitkraken.com/